scripts/1pass.sh

```diff
@@ -1,229 +0,0 @@
-#!/bin/bash
-
-
-# =================================================================================================================
-# Usage:
-# -----------------------------------------------------------------------------------------------------------------
-usage() {
```

Comment:
This still useful for OCP?

Reply:
https://github.com/bcgov/sbc-auth/blob/f5f83f1e05c633e8829ff8571e40000f3a58a9b9/auth-api/Makefile#L113